stdlib/stdlib.mli

```diff
@@ -13,15 +13,21 @@
 (*                                                                        *)
 (**************************************************************************)
 
-(** The initially opened module.
+(** The OCaml Standard library.
 
-   This module provides the basic operations over the built-in types
-   (numbers, booleans, byte sequences, strings, exceptions, references,
-   lists, arrays, input-output channels, ...).
+    This module is automatically opened at the beginning of each
+    compilation. All components of this module can therefore be
+    referred by their short name, without prefixing them by [Stdlib].
+*)
+
+module Pervasives : sig
+(** Pervasives operations.
```

Comment:
This should be `Pervasive operations.`, but I think it would be better for it be the same as old documentation and be `The initially opened module` (as this is still true)